### PR TITLE
Update staging/README.md

### PR DESCRIPTION
**Pull request by ialidzhikov**
/kind cleanup
/kind documentation

This syncs the `staging/README.md` syncs the staging repos with the current one:
- it seems that `k8s.io/component-helpers` was recently added
- there is no longer a staging pkg for `k8s.io/csi-api`

#### Does this PR introduce a user-facing change?

```release-note
NONE
```

#### Additional documentation e.g., KEPs (Kubernetes Enhancement Proposals), usage docs, etc.:

```docs

```

### CODE REVIEW

File: staging/README.md [hunk 1]
```diff
@@ -16,9 +16,9 @@ Repositories currently staged here:
 - [`k8s.io/cluster-bootstrap`](https://github.com/kubernetes/cluster-bootstrap)
 - [`k8s.io/code-generator`](https://github.com/kubernetes/code-generator)
 - [`k8s.io/component-base`](https://github.com/kubernetes/component-base)
+- [`k8s.io/component-helpers`](https://github.com/kubernetes/component-helpers)
 - [`k8s.io/controller-manager`](https://github.com/kubernetes/controller-manager)
 - [`k8s.io/cri-api`](https://github.com/kubernetes/cri-api)
-- [`k8s.io/csi-api`](https://github.com/kubernetes/csi-api)
```
Comment (astraw99) [inline staging/README.md:21]: Why delete this line?

Comment (ialidzhikov) [reply staging/README.md:21]: Why to keep it when the staging pkg does no longer exist?

**Pull request merged**